tests/test_config_parsing/config_dict_generator.py

```diff
@@ -389,6 +389,8 @@ def to_config_file(filename, config_dict):  # pylint: disable=too-many-branches
             if keyword == ConfigKeys.DATA_KW_KEY:
                 for define_key, define_value in keyword_value.items():
                     config.write(f"{keyword} {define_key} {define_value}\n")
+            elif keyword == pytest.TEST_CONFIG_FILE_KEY:
```

Comment:
I suggest we inline `pytest.TEST_CONFIG_FILE_KEY`and just write `keyword == "TEST_CONFIG_FILE"` everywhere.

Reply:
i'm not entirely sure what you mean - instead of holding the value in a variable, you want to repeat the string literal everywhere it's used?

Reply:
Yes, that was generally the idea. Since it is just used in tests that seemed unproblematic.

Reply:
Actually, I got a better idea, so I am closing this conversation.

Reply:
this is outdated now, the keyword is gone : )